lighthouse-core/report/v2/report-template.html

```diff
@@ -21,14 +21,15 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1">
   <title>Lighthouse Report</title>
+  <style>%%LIGHTHOUSE_CSS%%</style>
 </head>
 <body>
   <noscript>Lighthouse report requires JavaScript. Please enable.</noscript>
   <script>window.__LIGHTHOUSE_JSON__ = %%LIGHTHOUSE_JSON%%</script>
   <script>%%LIGHTHOUSE_JAVASCRIPT%%</script>
   <script>
-    const renderer = new ReportRenderer(window.__LIGHTHOUSE_JSON__);
-    renderer.render(document.body);
+    const renderer = new ReportRenderer(document);
+    document.body.appendChild(renderer.renderReport(window.__LIGHTHOUSE_JSON__));
```

Comment:
Hopefully in the near future, we can move some of the UI into static HTML that gets populated. Is there a plan for that? I'm still very concerned about the maintainability and scalability of a pure dom-based approach.